components/organisms/Layout.js

```diff
@@ -20,22 +22,50 @@ export const Layout = ({
   locale,
   langUrl,
 }) => {
+  //State for Search input
+  const [searchText, setSearchText] = useState(null);
+  //Handler for search input
+  function handleInputChange(e) {
+    setSearchText(e.target.value);
+  }
   const { t } = useTranslation("common");
   const language = locale === "en" ? "fr" : "en";
+
   return (
     <div className="overflow-x-hidden">
       <header>
         <PhaseBanner phase={t("Alpha")}>{t("alphaText")}</PhaseBanner>
-        <div className="layout-container">
-          <Link key={language} href={langUrl} locale={language}>
-            <a data-cy="toggle-language-link">
-              {language === "en" ? "English" : "Français"}
-            </a>
-          </Link>
-          <Header
-            headerLogoAltText="Symbol of the Government of Canada"
-            headerLogoImage="/sig-blk-en.svg"
-          />
+        <div className="layout-container flex-col flex lg:flex lg:flex-row justify-between  mt-2">
+          <div className="flex flex-row justify-between">
+            <div className=" lg:mt-7 mt-1.5 ">
```

Comment:
lots of divs here again

Reply:
line 40 if I removed the div the FR element for the translation would leave the layout-container and won't align correctly. Line 38 is for it to be in a row and and a column for the search bar. 
Line 39 works when the screen gets smaller and the link element will appear when visible tag is set. It will show the header and link tag in a row. Those I can not removed or I will need to redo the design.